src/burnchains/burnchain.rs

```diff
@@ -558,12 +529,17 @@ impl Burnchain {
         first_block_height: u64,
         first_block_hash: &BurnchainHeaderHash,
     ) -> Burnchain {
-        let mut ret = Burnchain::new(
-            &"/unit-tests".to_string(),
-            &"bitcoin".to_string(),
-            &"mainnet".to_string(),
-        )
-        .unwrap();
+        use rand::rngs::ThreadRng;
+        use rand::thread_rng;
+        use rand::RngCore;
+
+        let mut rng = thread_rng();
+        let mut byte_tail = [0u8; 16];
+        rng.fill_bytes(&mut byte_tail);
+
+        let tmp_path = format!("/tmp/unit-tests-{}", &to_hex(&byte_tail));
```

Comment:
Can this use something like `/tmp/stacks-node-tests/unit-tests-{}`? Most of the other tests at this point all use subdirectories of `/tmp/stacks-node-tests`

Reply:
Yup, fixed